pkg/remote/deploy_pfe.go

```diff
@@ -47,12 +47,12 @@ func DeployPFE(config *restclient.Config, clientset *kubernetes.Clientset, codew
 	}
 
 	logr.Infof("Checking if '%v' role bindings exist\n", codewindRoleBindingName)
-	rolebindings, err := clientset.RbacV1().ClusterRoleBindings().Get(codewindRoleBindingName, metav1.GetOptions{})
+	rolebindings, err := clientset.RbacV1().RoleBindings(codewindInstance.Namespace).Get(codewindRoleBindingName, metav1.GetOptions{})
 	if rolebindings != nil && err == nil {
-		logr.Warnf("Cluster role binding '%v' already exist.\n", codewindRoleBindingName)
+		logr.Warnf("Role binding '%v' already exist.\n", codewindRoleBindingName)
```

Comment:
There is no warn level in the switch at the bottom of `commands.go` so would be good to add it for this